extensions/ql-vscode/src/qltest-discovery.ts

```diff
@@ -128,12 +140,16 @@ export class QLTestDiscovery extends Discovery<QLTestDiscoveryResults> {
   /**
    * Event to be fired when the set of discovered tests may have changed.
    */
-  public get onDidChangeTests(): Event<void> { return this._onDidChangeTests.event; }
+  public get onDidChangeTests(): Event<void> {
+    return this._onDidChangeTests.event;
+  }
 
   /**
    * The root test directory for each QL pack that contains tests.
```

Comment:
Update - I assume this is just the workspace folder, viewed as a `QLTestDirectory`.